project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject com.roomkey/lein-v :lein-v
+(defproject com.roomkey/lein-v "0.0.0"
```

Comment:
Why is this required?  Last I checked, `lein-v` was self-aware...  but I am aware of issues when backwards compatibility is not provided.  Can you fill me in on your specific issue?

Reply:
Ok, that one turns out to be "necessary" because under Cursive/IntelliJ (and under some conditions in leiningen, e.g. when running "lein check") the project.clj file cannot be read with a keyword as "version". ("clojure.lang.Keyword cannot be cast to java.lang.String", see below)

But you're right, I just changed this ... but it's not essentially a part of the feature of this merge request.

So,  it's up to you to decide whether you want it or leave it.

Stack trace on `lein check`:

```
java.lang.ClassCastException: clojure.lang.Keyword cannot be cast to java.lang.String
 at leiningen.core.project$make_project_properties.invokeStatic (project.clj:19)
    leiningen.core.project$make_project_properties.invoke (project.clj:16)
    leiningen.core.eval$write_pom_properties.invokeStatic (eval.clj:41)
    leiningen.core.eval$write_pom_properties.invoke (eval.clj:36)
    leiningen.core.eval$prep.invokeStatic (eval.clj:84)
    leiningen.core.eval$prep.invoke (eval.clj:73)
    leiningen.core.eval$eval_in_project.invokeStatic (eval.clj:362)
    leiningen.core.eval$eval_in_project.invoke (eval.clj:356)
    leiningen.core.eval$eval_in_project.invokeStatic (eval.clj:360)
    leiningen.core.eval$eval_in_project.invoke (eval.clj:356)
    leiningen.check$check.invokeStatic (check.clj:32)
    leiningen.check$check.invoke (check.clj:8)
```

Reply:
I was unable to reproduce that failure... and then I remembered why.  In order to "eat our own dog food", I have a `.lein-classpath` file in the root of the project.  That _should_ allow `lein check` to work if you have a tolerably recent version of `lein-v` installed locally.

But it *must* be installed.  Can you try installing a recent version of `lein-v` locally and let me know if you can then run with the keyword?  I really would like to continue being "self aware" on this front.

Reply:
oh, this does not help with IntelliJ, and I had lein-v installed locally... so, no, does not work for me. I'm using lein 2.7.1.

Reply:
left this out of the PR #10.